app/uk/gov/hmrc/transitmovements/services/DepartureFactory.scala

```diff
@@ -37,6 +38,7 @@ trait DepartureFactory {
 
   def create(
     eori: EORINumber,
+    movementType: MovementType,
```

Comment:
The movement type will always be a `MovementType.Departure` here - it's the `DepartureFactory`!

Reply:
This will be renamed to MovementFactory, which supports both Arrival and Departure